spec/models/lists/establishment_spec.rb

```diff
@@ -62,5 +38,37 @@ module Lists
         end
       end
     end
+
+    describe 'uses links correctly' do
```

Comment:
I think you could make this a bit easier to read by removing the hard-coded ids and creating the relationships in the assertions, e.g.

```
it 'finds successor' do
  establishment = create(:closed_establishment)
  successor = create(:establishment_link_successor, establishment: establishment)
  expect(establishment.successor).to eq(successor)
end
```
You could also add another `link_type` and confirm successor find the right relationships.